inlang/source-code/sdk2/src/lix-plugin/reportConflicts.test.ts

```diff
@@ -0,0 +1,183 @@
+/* eslint-disable @typescript-eslint/no-non-null-assertion */
+import { test, expect } from "vitest";
+import { inlangLixPluginV1 } from "./inlangLixPluginV1.js";
+import { newLixFile, openLixInMemory, type Change } from "@lix-js/sdk";
+
+test("a create operation should not report a conflict given that the change does not exist in target", async () => {
+	const targetLix = await openLixInMemory({ blob: await newLixFile() });
+	const sourceLix = await openLixInMemory({ blob: await newLixFile() });
```

Comment:
I used `target` and `source` as terms in the hope that `target` is signalling "aha we are copy changes into the target". 

@janfjohannes what do you think about the terms?

Reply:
sounds clear and good. we need similar good terms instead of old and neu asap

Reply:
I opened an issue for old/neu https://github.com/opral/lix-sdk/issues/44